### PR TITLE
COMMERCE-11706

### PR DESCRIPTION
**Pull request by Brunno0liveira**
[COMMERCE-11706](https://issues.liferay.com/browse/COMMERCE-11706)

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 1]
```diff
@@ -1008,6 +1008,127 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11706. Verify that only one / is shown in the URLs inside the sitemap when Virtual Host is enabled"
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase:1011]: Verify that only one "/" is shown in the URLs inside the sitemap when Virtual Host is enabled.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 2]
```diff
@@ -1008,6 +1008,127 @@ definition {
...
+
+			Select(
+				locator1 = "Select#ANY_ID",
+				selectId = "friendly-url-style",
+				value1 = "Locale is automatically prepended to every URL.");
+
+			PortalSettings._saveDisplaySettings();
+		}
+
+		task ("Given an account is created") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase:1037]: And an account is created

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 3]
```diff
@@ -1008,6 +1008,127 @@ definition {
...
+
+			Type(
+				key_fieldLabel = "Friendly URL",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "sensor-abs");
+
+			Button.clickPublish();
+		}
+
+		task ("Define public virtual host and private virtual host") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase:1081]: And public virtual host and private virtual host are defined

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 4]
```diff
@@ -1008,6 +1008,127 @@ definition {
...
+			Site.openSiteSettingsAdmin(siteURLKey = "Minium");
+
+			Site.addVirtualHostsURLCP(
+				pageVirtualHosts = "true",
+				pageVirtualHostURL = "www.able.com");
+		}
+
+		task ("Sign in the page virtual host") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase:1089]: And the virtual host signs in to the page

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase [hunk 5]
```diff
@@ -1008,6 +1008,127 @@ definition {
...
+			Open.openNoErrorNoSPARefresh(locator1 = ${link});
+		}
+
+		task ("Then the URLs are shown with only one /") {
+			AssertElementPresent.assertElementPresentNoWaitForSPAReFresh(locator1 = "//*[text()='http://www.able.com:8080/ca/p/sensor-abs']");
+
+			AssertElementPresent.assertElementPresentNoWaitForSPAReFresh(locator1 = "//*[text()='http://www.able.com:8080/en/p/abs-sensor']");
+		}
+
+		task ("And the admin goes back to portal") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductConfigurations.testcase:1125]: Why does the admin need to go back to the portal?

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look. Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
@victoroscuro No related errors to this pull, it can be force-forwarded, thanks!

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**